Configuring storybook to an angular library breaks Module Federation

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
I have an Nx solution that has 5 applications (1 host, and 4 remote sites) and they all work fine.
`nx build` is successful
`nx run host:serve-mfe` is successful

## Expected Behavior
`yarn nx generate @nrwl/angular:storybook-configuration --name=shared-core` should not break module federation

## Steps to Reproduce

- Follow all the steps in the [Nx Module Federation Guide](https://nx.dev/guides/setup-mfe-with-angular#how-to-setup-a-micro-frontend-with-angular-and-nx)
- Confirm that it is working as expected `nx run dashboard:serve-mfe`
- add storybook configuration: `nx generate @nrwl/angular:storybook-configuration --name=shared-data-access-user`
-  Confirm that it is still working as expected `nx run dashboard:serve-mfe`


### Failure Logs
nx run dashboard:serve-mfe

[nx serve dashboard] 
[nx serve dashboard] > nx run dashboard:serve:development

[nx serve login]
[nx serve login] > nx run login:serve:development
[nx serve login]
[nx serve login] - Generating browser application bundles (phase: setup)...
[nx serve dashboard] - Generating browser application bundles (phase: setup)...
[nx serve login] Cannot find module 'webpack/lib/container/ModuleFederationPlugin'
Require stack:
- C:\code\ng-mfe\apps\login\webpack.config.js
- C:\code\ng-mfe\node_modules\@nrwl\angular\src\builders\utilities\webpack.js
- C:\code\ng-mfe\node_modules\@nrwl\angular\src\builders\webpack-server\webpack-server.impl.js
- C:\code\ng-mfe\node_modules\@angular-devkit\architect\node\node-modules-architect-host.js
- C:\code\ng-mfe\node_modules\@angular-devkit\architect\node\index.js
- C:\code\ng-mfe\node_modules\@nrwl\tao\src\commands\ngcli-adapter.js
- C:\code\ng-mfe\node_modules\@nrwl\tao\src\commands\run.js
- C:\code\ng-mfe\node_modules\@nrwl\tao\index.js
- C:\code\ng-mfe\node_modules\@nrwl\cli\lib\run-cli.js
[nx serve dashboard] Cannot find module 'webpack/lib/container/ModuleFederationPlugin'
Require stack:
- C:\code\ng-mfe\apps\dashboard\webpack.config.js
- C:\code\ng-mfe\node_modules\@nrwl\angular\src\builders\utilities\webpack.js
- C:\code\ng-mfe\node_modules\@nrwl\angular\src\builders\webpack-server\webpack-server.impl.js
- C:\code\ng-mfe\node_modules\@angular-devkit\architect\node\node-modules-architect-host.js
- C:\code\ng-mfe\node_modules\@angular-devkit\architect\node\index.js
- C:\code\ng-mfe\node_modules\@nrwl\tao\src\commands\ngcli-adapter.js
- C:\code\ng-mfe\node_modules\@nrwl\tao\src\commands\run.js
- C:\code\ng-mfe\node_modules\@nrwl\tao\index.js
- C:\code\ng-mfe\node_modules\@nrwl\cli\lib\run-cli.js
Warning: @nrwl/run-commands command "nx serve login" exited with non-zero status codeWarning: @nrwl/run-commands command "nx serve dashboard" exited with non-zero status 
code
 ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————— 

 >  NX   Running target "dashboard:serve-mfe" failed

   Failed tasks:

   - dashboard:serve-mfe

### Environment
 Node : 16.13.1
   OS   : win32 x64
   yarn : 1.22.17

   nx : 13.8.3
   @nrwl/angular : 13.8.3
   @nrwl/cli : 13.8.3
   @nrwl/cypress : 13.8.3
   @nrwl/detox : undefined
   @nrwl/devkit : 13.8.3
   @nrwl/eslint-plugin-nx : 13.8.3
   @nrwl/express : undefined
   @nrwl/jest : 13.8.3
   @nrwl/js : undefined
   @nrwl/linter : 13.8.3
   @nrwl/nest : undefined
   @nrwl/next : undefined
   @nrwl/node : undefined
   @nrwl/nx-cloud : undefined
   @nrwl/react : undefined
   @nrwl/react-native : undefined
   @nrwl/schematics : undefined
   @nrwl/storybook : 13.8.3
   @nrwl/tao : 13.8.3
   @nrwl/web : undefined
   @nrwl/workspace : 13.8.3
   typescript : 4.5.5
   rxjs : 7.4.0
   ---------------------------------------
   Community plugins:
         @angular-architects/module-federation: 13.0.1
         @angular/animations: 13.2.4
         @angular/common: 13.2.4
         @angular/compiler: 13.2.4
         @angular/core: 13.2.4
         @angular/forms: 13.2.4
         @angular/platform-browser: 13.2.4
         @angular/platform-browser-dynamic: 13.2.4
         @angular/router: 13.2.4
         @angular-devkit/build-angular: 13.2.5
         @angular/cli: 13.2.5
         @angular/compiler-cli: 13.2.4
         @angular/language-service: 13.2.4
         @storybook/angular: 6.4.19

(node:12216) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at C:\code\ng-mfe\node_modules\tslib\package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)

--- a/packages/storybook/src/generators/init/init.ts
+++ b/packages/storybook/src/generators/init/init.ts
@@ -18,6 +18,7 @@ import {
   storybookVersion,
   svgrVersion,
   urlLoaderVersion,
+  webpack5Version,
 } from '../../utils/versions';
 import { Schema } from './schema';
 
@@ -63,6 +64,7 @@ function checkDependenciesInstalled(host: Tree, schema: Schema) {
       !packageJson.devDependencies['@storybook/manager-webpack5']
     ) {
       devDependencies['@storybook/manager-webpack5'] = storybookVersion;
+      devDependencies['webpack'] = webpack5Version;
     }
 
     if (

--- a/packages/storybook/src/utils/versions.ts
+++ b/packages/storybook/src/utils/versions.ts
@@ -5,6 +5,6 @@ export const babelLoaderVersion = '8.1.0';
 export const babelPresetTypescriptVersion = '7.12.13';
 export const svgrVersion = '^5.4.0';
 export const urlLoaderVersion = '^3.0.0';
-
+export const webpack5Version = '^5.64.0';
 export const storybookReactNativeVersion = '6.0.1-alpha.7';
 export const reactNativeStorybookLoader = '^2.0.5';

